src/lib/api/projects.ts

```diff
@@ -62,6 +64,67 @@ export async function list(
   return res.json();
 }
 
+/**
+ * Get a list of all projects owned by the user
+ */
+export async function getOwned(
+  userId: number,
+  query?: string,
+): Promise<Project[]> {
```

Comment:
This should take a `fetch` parameter so we can use SvelteKit's fetch wrapper, and for testing.